src/games/strategy/engine/framework/mapDownload/DownloadRunnable.java

```diff
@@ -80,4 +106,15 @@ public void run() {
       }
     }
   }
+
+  private void readLocalFile() {
```

Comment:
New read local file method, simply we assume that we're reading a file relative the root folder. Use nio Files.readAllBytes to read the file data.